packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -586,21 +606,19 @@ class _UiKitViewState extends State<UiKitView> {
 
   @override
   Widget build(BuildContext context) {
-    final UiKitViewController? controller = _controller;
+    final TController? controller = _controller;
     if (controller == null) {
       return const SizedBox.expand();
     }
     return Focus(
       focusNode: focusNode,
       onFocusChange: (bool isFocused) => _onFocusChange(isFocused, controller),
-      child: _UiKitPlatformView(
-        controller: _controller!,
-        hitTestBehavior: widget.hitTestBehavior,
-        gestureRecognizers: widget.gestureRecognizers ?? _emptyRecognizersSet,
-      ),
+      child: childPlatformView()
     );
   }
 
+  TPrivateView childPlatformView();
```

Comment:
why is it called private view?

Reply:
This template class corresponds to the private view class used by the subclass, e.g. `_UiKitPlatformView`, as opposed to the public class `UiKitView`.

Reply:
Looks like Google's style convention, at least in Java, is to append T at the end of long form generic types. I will update this